keyworker-specs/src/test/groovy/uk/gov/justice/digital/hmpps/keyworker/specs/MaintainRolesSpecification.groovy

```diff
@@ -157,7 +162,7 @@ class MaintainRolesSpecification extends GebReportingSpec {
         then:
         at UserSearchResultsPage
         rows.size() == 3
-        nextPage.text() == "Next\n3 of 3"
+        waitFor { nextPage.text() == "Next\n3 of 3" }
```

Comment:
this is the first bit of page content that will be different from the previous page so wait until it is different